test/functional/apps/visualize/_data_table.js

```diff
@@ -206,6 +206,11 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.visualize.toggleOtherBucket();
         await PageObjects.visualize.toggleMissingBucket();
         await PageObjects.visualize.clickGo();
+
+        await PageObjects.visualize.toggleOtherBucket();
+        await PageObjects.visualize.toggleMissingBucket();
+        await PageObjects.visualize.clickGo();
```

Comment:
I noticed the test is failing on CI so you may change this anyway, but it might be worth adding some sort of comment here with an explanation for the duplication or link to the relevant issue.

If I were to stumble across this test on my own, I would likely remove the duplicated lines, assuming it was a typo.